controller/handler/trial-csv.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * @module controller/handler/trial-csv
+ */
+
+const database = require('../../model');
+const convertJsonToCsv = require('../helper/convert-json-to-csv');
+const boom = require('boom');
+const deduplicate = require('../helper/deduplicate');
+const configuration = [
+    {
+        label: 'patient pin',
+        key: 'pin',
+        default: 'DNE'
+    },
+    {
+        label: 'survey name',
+        key: 'name'
+    },
+    {
+        label: 'unique survey id',
+        key: 'id'
+    },
+    {
+        label: 'unique question id',
+        key: 'questionId'
+    },
+    {
+        label: 'question',
+        key: 'questionText'
+    },
+    {
+        label: 'question option',
+        key: 'optionText'
+    }
+];
+
+/**
+ * Create a Comma Seperate Value export of a single patient's data.
+ * @param {Request} request - Hapi request
+ * @param {Reply} reply - Hapi Reply
+ * @returns {View} Rendered page
+ */
+function trialCSV (request, reply) {
+    database.sequelize.query(
+        `
+        SELECT pa.pin, st.name, si.id, qt.id AS questionId, qt.questionText, qo.id AS optionId, qo.optionText
```

Comment:
we may want patient more details about the patient to be exported. (such as start, end, and device used)

Thoughts @promisasu/admin ?